frontend/src/UploadForm.js

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+import React, {useState} from 'react';
+
+export default function UploadForm() {
+	const [selectedFile, setSelectedFile] = useState();
+	const [isSelected, setIsSelected] = useState(false);
+
+	function handleFileChange(event) {
+		setSelectedFile(event.target.files[0]);
+		setIsSelected(true);
+	};
+
+	async function handleFileSubmit() {
+		const form = new FormData();
+		form.append('file', selectedFile);
+
+        const response = await axios.post("url", form, {
```

Comment:
What if the response is not successful?

Reply:
Just leave it there.

Currently this POST request is dummy implementation and will never work properly (as you see it is sending the request to "url" which is not an actual URL). Once upload functionality in the backend is finished we can complete this part.